packages/astroport-governance/src/assembly.rs

```diff
@@ -31,58 +31,58 @@ pub struct InstantiateMsg {
     pub proposal_required_threshold: String,
 }
 
+/// # Description
```

Comment:
Let's not use such headings on items because it reduces documentation readability, e. g. https://docs.rs/astroport/1.0.1/astroport/asset/index.html

Reply:
Removed all "## Description" headings from packages/